packages/analytics-js-plugins/src/iubendaConsentManager/index.ts

```
@@ -0,0 +1,119 @@
+          }
+
+          const configuredConsents = cmpConfig.consents.map(c => c.consent.trim()).filter(n => n);
+          const resolutionStrategy =
+            cmpConfig.resolutionStrategy ?? state.consents.resolutionStrategy.value;
+  
+          // match the configured consents with user provided consents as per
+          // the configured resolution strategy
+          switch (resolutionStrategy) {
+            case 'or':
+              return configuredConsents.some(matchPredicate) || configuredConsents.length === 0;
+            case 'and':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)